### DISCUSSION TITLE
💡 Feature Request: Treasure room

### DISCUSSION THREAD

**Discussion by Dx8pi** [DiscordDungeons/Bugs#1332] [Ideas]
### Is your feature request related to a problem?

1/1000 chance are you fucking insane. Medal of a god all over again

### Describe alternatives you've considered

1/50 chance, no need to clear all enemies on a floor to get it.

### 📝 Additional context

Enemies in dungeons should be reconsidered, their only real use right now is slowly draining the players health by escaping form them, maybe keep building on that concept. Convert enemies to, lets say booby traps, "You narrowly escaped the trap" "You got hit and lost X hp!" or thorny bushes, whatever, no one genuinely fights the enemies anyways and those who do are stupid. I rest my case.

**Comment by Willard21** [comment]
Don't worry, you don't actually have to defeat all the enemies to get a treasure room. It checks for enemies before generating the level, so it's always "cleared" by default lol
